FindYourMovie/Controllers/MovieSearchController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  MovieSearchController.swift
+//  FindYourMovie
+//
+//  Created by Andres Guevara Caprio on 8/19/20.
+//  Copyright © 2020 Andres Guevara Caprio. All rights reserved.
+//
+
+import UIKit
+
+/// Controller used when the user is going to search for movies. This is a Table View that displays the last 10 searches made by the user.
+class MovieSearchController: UITableViewController {
+
+    var searches = [String]()
+    var parentController : MovieViewController? = nil
```

Comment:
when declaring an optional like this, there is no need to assign nil to it, as it will be nil at creation. In some respects this is a style critique (which I try to avoid in these PRs), but wanted to call this out.
`var parentController: MovieViewController?` is adequate.